hosts/testagent/brainstem.nix

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
Considering it's vendored in from nixpkgs, needs an MIT SPDX Header.

Reply:
FYI: some files in this project already use MIT license, e.g.: [here](https://github.com/tiiuae/ghaf-infra/blob/main/tasks.py#L3-L5) and [here](https://github.com/tiiuae/ghaf-infra/blob/main/shell.nix#L1-L3)